### ISSUE TITLE
Cross-Validation (full section)

### ISSUE THREAD

**Issue by SamLau95** [DS-100/textbook#14]
I would like a section on CV at the end of chapter 13 on Feature Engineering. This will be section 13.3 which comes right after [section 13.2](https://www.textbook.ds100.org/ch13/feature_polynomial.html) on polynomial regression.

**Outline:**

- In the previous section, we see that adding many features to the data will lower cost but result in poor models. (Show short example.)
- How do we choose the best set of features for the data?
  - We need to evaluate the model based on data that it hasn't used for training.
- Cross validation
  - We split the data into a training set, validation set, and test set.
  - We train the model on the training set, then check its accuracy on the validation set.
  - Show example using ice cream regression from last section
  - We use the training set for fitting the model; we use CV for model selection.
- Test set accuracy
  - Still, the validation set accuracy will usually be too high since we picked the model with the highest validation accuracy.
  - So, we use the test set (which the model has never seen at all) to report the final accuracy of the model. We cannot use the test set accuracy to pick a different model.
  - Show test set accuracy on ice cream regression
- Conclusion: We use CV to perform feature selection and model selection.

**Comment by SamLau95** [comment]
Done! https://www.textbook.ds100.org/ch13/feature_cv.html
